runtime/service.go

```diff
@@ -598,44 +600,63 @@ func (s *service) startVM(ctx context.Context, request *taskAPI.CreateTaskReques
 	}
 
 	cfg := firecracker.Config{
-		BinPath:         s.config.FirecrackerBinaryPath,
 		SocketPath:      s.config.SocketPath,
 		VsockDevices:    []firecracker.VsockDevice{{Path: "root", CID: cid}},
 		KernelImagePath: s.config.KernelImagePath,
 		KernelArgs:      s.config.KernelArgs,
-		RootDrive:       firecracker.BlockDevice{HostPath: s.config.RootDrive, Mode: "rw"},
-		CPUCount:        int64(s.config.CPUCount),
-		CPUTemplate:     firecracker.CPUTemplate(s.config.CPUTemplate),
-		MemInMiB:        256,
-		Console:         s.config.Console,
```

Comment:
Since Console was removed here, can it safely be removed from runtime/config.go Config struct?

Reply:
Removed.